build_tools/fbcode_config4.8.1.sh

```diff
@@ -93,7 +93,7 @@ else
 fi
 
 CFLAGS+=" $DEPS_INCLUDE"
-CFLAGS+=" -DROCKSDB_PLATFORM_POSIX -DROCKSDB_LIB_IO_POSIX -DROCKSDB_FALLOCATE_PRESENT -DROCKSDB_MALLOC_USABLE_SIZE -DROCKSDB_RANGESYNC_PRESENT -DROCKSDB_SCHED_GETCPU_PRESENT -DROCKSDB_SUPPORT_THREAD_LOCAL"
+CFLAGS+=" -DROCKSDB_PLATFORM_POSIX -DROCKSDB_LIB_IO_POSIX -DROCKSDB_FALLOCATE_PRESENT -DROCKSDB_MALLOC_USABLE_SIZE -DROCKSDB_RANGESYNC_PRESENT -DROCKSDB_SCHED_GETCPU_PRESENT -DROCKSDB_SUPPORT_THREAD_LOCAL -DHAVE_SSE42"
```

Comment:
why do we need it here as it's already added in build_detect_platform when compiling sse4.2 instructions succeeds?

Reply:
I don't think it's correct to have it hard coded there. In build_detect_platform we do test compiling and set -DHAVE_SSE42 only when SSE4.2 is really supported. On a system that does not support SSE42 if -DHAVE_SSE42 is hard set here then it may execute wrong code